openstf/model/fallback.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from pathlib import Path
+from datetime import datetime, timedelta, timezone
+
+import pandas as pd
+import structlog
+
+
+def generate_fallback(
+    forecast_input: pd.DataFrame,
+    load: pd.DataFrame,
+    fallback_strategy: str = "extreme_day",
+):
```

Comment:
Nice that you are using type hints here, you should howere also add a type hint for the output. Suggestion:
def generate_fallback(
    forecast_input: pd.DataFrame,
    load: pd.DataFrame,
    fallback_strategy: str = "extreme_day",
) -> pd.DataFrame :

Reply:
Good suggestion, I'll add it